docs/user-guide/connecting-applications.md

```diff
@@ -183,19 +183,16 @@ nginxsecret           Opaque                                2
 
 Now modify your nginx replicas to start an https server using the certificate in the secret, and the Service, to expose both ports (80 and 443):
 
-{% include code.html language="yaml" file="nginx-secure-app.yaml" ghlink="/docs/user-guide/nginx-secure-app" %}
+{% include code.html language="yaml" file="nginx-secure-app.yaml" ghlink="/docs/user-guide/nginx-secure-app.yaml" %}
 
 Noteworthy points about the nginx-secure-app manifest:
 
-- It contains both rc and service specification in the same file
+- It contains both Deployment and Service specification in the same file
```

Comment:
Did you mean /docs/user-guide/nginx-secure-app.yaml? It uses a Deployment already